lightning/src/ln/functional_tests.rs

```diff
@@ -8122,7 +8122,7 @@ fn test_manually_accept_inbound_channel_request() {
 	let events = nodes[1].node.get_and_clear_pending_events();
 	match events[0] {
 		Event::OpenChannelRequest { temporary_channel_id, .. } => {
-			nodes[1].node.accept_inbound_channel(&temporary_channel_id).unwrap();
+			nodes[1].node.accept_inbound_channel(&temporary_channel_id, None).unwrap();
```

Comment:
Should be trivial to just set a value here and test that you get it back if the channel is closed, no? Given its trivial might as well add a test to ensure we don't regress...you never know :p